pkgs/by-name/cp/cpplint/package.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  versionCheckHook,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "cpplint";
+  version = "2.0.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "cpplint";
+    repo = "cpplint";
+    rev = "refs/tags/${version}";
+    hash = "sha256-076363ZwcriPb+Fn9S5jay8oL+LlBTNh+IqQRCAndRo=";
+  };
+
+  postPatch = ''
+    substituteInPlace setup.cfg \
+      --replace-fail "pytest-cov" "" \
+      --replace-fail "--cov-fail-under=90 --cov=cpplint" ""
```

Comment:
We now have the pytest-cov-stub package which we can just add instead

Reply:
I tried the following:

1) Adding `pytest-cov-stub` (and not `pytest-cov`):
```
Executing pytestCheckPhase
ERROR: Missing required plugins: pytest-cov
```

2) Having both `pytest-cov` and `pytest-cov-stub`:
The tests do run but still fail at the end with
```
FAIL Required test coverage of 90% not reached. Total coverage: 89.56%
```

Is there something I missed ?